src/configurations/destinations/mp/ui-config.json

```diff
@@ -1,448 +1,531 @@
+                    "required": true,
+                    "default": "Viewed {{ category }} {{ name }} page",
+                    "note": "Assign a event name for your page calls. Text {{ }} will be replaced with event payload value",
+                    "preRequisites": {
+                      "fields": [
+                        {
+                          "configKey": "useUserDefinedPageEventName",
+                          "value": true
+                        }
+                      ]
+                    }
+                  }
+                ]
+              },
+              {
+                "title": "Screen settings",
+                "note": "Set how you want to send your page calls to Mixpanel",
+                "icon": "file",
+                "fields": [
+                  {
+                    "type": "checkbox",
+                    "label": "Use Custom Screen Event Name",
+                    "configKey": "useUserDefinedScreenEventName",
+                    "note": "Send custom event name in screen call",
+                    "default": false
+                  },
+                  {
+                    "type": "textInput",
+                    "label": "Screen Event Name Template",
+                    "configKey": "userDefinedScreenEventTemplate",
+                    "regex": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{0,200})$",
+                    "regexErrorMessage": "Invalid Screen Event Name",
+                    "required": true,
+                    "default": "Viewed {{ category }} {{ name }} screen",
+                    "note": "Assign a event name for your screen calls. Text {{ }} will be replaced with event payload value",
+                    "preRequisites": {
+                      "fields": [
+                        {
+                          "configKey": "useUserDefinedScreenEventName",
+                          "value": true
+                        }
+                      ]
+                    }
+                  }
+                ]
+              },
+              {
+                "title": "Mixpanel Configuration Settings",
+                "fields": [
+                  {
+                    "type": "checkbox",
+                    "label": "Strict Mode",
+                    "configKey": "strictMode",
+                    "default": false,
+                    "note": "If enabled, Mixpanel will validate the request and return errors per event that failed",
+                    "preRequisites": {
+                      "fields": [
+                        {
+                          "configKey": "connectionModes.cloud",
+                          "value": true
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "type": "tagInput",
+                    "configKey": "propIncrements",
+                    "label": "Properties to increment in People",
```

Comment:
Is propIncrements applicable only for web device mode ?

Reply:
`propIncrements` is available for both cloud and device mode.